runtime/doc/starting.txt

```diff
@@ -381,76 +350,6 @@ argument.
 							*-W*
 -W {scriptout}	Like -w, but do not append, overwrite an existing file.
 
---remote [+{cmd}] {file} ...
-		Open the {file} in another Vim that functions as a server.
-		Any non-file arguments must come before this.
-		See |--remote|.
```

Comment:
I removed these because nvim doesn't have the options, that's all. It may make our lives ever so slightly easier when we want to implement this, but this documentation is probably of no use to the majority of users.

If anything it's misleading because there's no indication in these docs that Nvim doesn't have these options.